src/main/java/org/broadinstitute/hellbender/utils/recalibration/RecalibrationReport.java

```diff
@@ -64,18 +74,79 @@ public RecalibrationReport(final GATKReport report, final SortedSet<String> allR
     }
 
     /**
-     * Gets the unique read groups in the table
+     * Gather multiple {@link RecalibrationReport}s into a single file
+     * @param inputs a list of {@link RecalibrationReport} files to gather
+     * @param output a file to write the recalibration reports to
+     */
+    public static void gatherReportsIntoOneFile(final List<File> inputs, final File output) {
+        Utils.nonNull(inputs, "inputs");
+        Utils.nonNull(output, "output");
+        try (final PrintStream outputFile = new PrintStream(output)){
+            final GATKReport report = gatherReports(inputs);
+            report.print(outputFile);
+        } catch(final FileNotFoundException e) {
+            throw new UserException.CouldNotCreateOutputFile(output, e);
+        }
+    }
+
+    /**
+     * Gathers a set of files containing {@link RecalibrationReport}s into a single {@link GATKReport}.
      *
-     * @param report the GATKReport containing the table with RecalUtils.READGROUP_REPORT_TABLE_TITLE
-     * @return the unique read groups
+     * @param inputs a list of files containing {@link RecalibrationReport}s
+     * @return gathered recalibration GATK report
      */
-    private static SortedSet<String> getReadGroups(final GATKReport report) {
-        final GATKReportTable reportTable = report.getTable(RecalUtils.READGROUP_REPORT_TABLE_TITLE);
-        final SortedSet<String> readGroups = new TreeSet<>();
-        for ( int i = 0; i < reportTable.getNumRows(); i++ ) {
-            readGroups.add(reportTable.get(i, RecalUtils.READGROUP_COLUMN_NAME).toString());
+    public static GATKReport gatherReports(final List<File> inputs) {
```

Comment:
So this currently does inputs.size() combines.  This is  a slow operation.  We could be faster if we implemented a tree merging strategy, but that was more complicated and I doubt this is the rate limiting step anywhere.